data/quarg/quarg missions.txt

```diff
@@ -325,3 +325,150 @@ mission "Quarg Pug Arfecta Warning"
 			`	The Quarg takes one last long look at your ship, then makes a curt farewell motion, leaving you to your business. The crowd, too, starts to disperse, with only a few staying behind to gawk at the ship.`
 			`	As you make your way through the ring, you notice occasional glares from the Quarg onboard the ring.`
 				decline
+
+
+
+mission "Ask Quarg About Coalition Late"
+	invisible
+	landing
+	deadline 1
+	to offer
+		has "Ask Quarg About Coalition Early: offered"
+		or
+			has "joined the heliarchs"
+			has "joined the lunarium"
+	source
+		attributes "quarg"
+		attributes "station"
+	on offer
+		conversation
+			branch heliarch
+				has "joined the heliarchs"
+			branch lunarium
+				has "joined the lunarium"
+			label heliarch
+			`You've obtained the rank of Heliarch, the group who to this day remain hostile to the Quarg. Despite that you were allowed to land here on <origin> - you don't know whether that's because the Quarg simply don't know that you've joined the Heliarchs, or don't care enough to stop you from coming to their rings. Would you like to ask the Quarg for their side of the story, in light of what you've learned from the Heliarchs?`
```

Comment:
```suggestion
			`You've obtained the rank of Heliarch, the group who to this day remain hostile to the Quarg. Despite that, you were allowed to land here on <origin> - you don't know whether that's because the Quarg simply don't know that you've joined the Heliarchs, or don't care enough to stop you from coming to their rings. Would you like to ask the Quarg for their side of the story, in light of what you've learned from the Heliarchs?`
```